it/uk/gov/hmrc/tai/integration/TaxAccountSummarySpec.scala

```diff
@@ -35,7 +35,8 @@ class TaxAccountSummarySpec extends IntegrationSpec {
   }
 
   val apiUrl = s"/tai/$nino/tax-account/$year/summary"
-  def request = FakeRequest(GET, apiUrl).withHeaders("X-SESSION-ID" -> generateSessionId)
+  def request = FakeRequest(GET, apiUrl)
+    .withHeaders("X-SESSION-ID" -> generateSessionId, "AUTHORIZATION" -> "Bearer 11")
```

Comment:
Use HeaderNames.xSessionId and HeaderNames.authorisation

Reply:
Updated